### PR TITLE
[makefile] make error message clearer with instructions

### PR DESCRIPTION
**Pull request by yxieca**
- What I did**
Make error message clearer. And give install command.

**- How to verify it**
Tested the error message by putting it outside the if block (since I already installed the j2 package).

### CODE REVIEW

File: Makefile.work [hunk 1]
```diff
@@ -48,7 +48,7 @@ endif
 # Check for j2cli availability
 J2_VER := $(shell j2 --version 2>&1 | grep j2cli | awk '{printf $$2}')
 ifeq ($(J2_VER),)
-$(error Install j2cli)
+$(error Please install j2 (sudo snap install j2))
```
Comment (jleveque) [inline Makefile.work]: Why snap and not `pip install j2cli`?

Comment (yxieca) [reply Makefile.work]: I tried apt-get install j2. And got prompt to install with snap install j2.

So pip install j2cli is the proper way?

Comment (lguohan) [reply Makefile.work]: snap only works for ubuntu, pip is better.

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**